htmresearch/frameworks/pytorch/linear_sdr.py

```diff
@@ -108,6 +114,24 @@ def __init__(self,
       self.rezeroWeights()
 
 
+  def normalizeWeights(self):
+    """
+    Initialize the weights using kaiming_uniform initialization normalized to
+    the number of non-zeros in the layer instead of the whole input size.
+    Similar to torch.nn.Linear.reset_parameters() but applying weight sparsity
+    to the input size
+    """
+    _, inputSize = self.l1.weight.shape
+    fan = int(inputSize * self.weightSparsity)
+    gain = nn.init.calculate_gain('leaky_relu', math.sqrt(5))
```

Comment:
Do you know what this gain ends up being usually?

Reply:
Here, using 'leaky_relu'  gain is always `math.sqrt(2/6)`
See https://pytorch.org/docs/stable/_modules/torch/nn/init.html